src/db/sqlite/lrsql/sqlite/sql/query.sql

```diff
@@ -327,3 +327,59 @@ WHERE id > :since-id
   AND id <= :until-id
 GROUP BY stored_time
 ORDER BY stored_time ASC;
+
+-- :name delete-actor-st2actor!
+-- :command :execute
+-- :result :affected
+DELETE FROM statement_to_actor where actor_ifi = :actor-ifi
```

Comment:
Why do you have DELETE statements in a query.sql file? I'm assuming you copy-pasted this here without looking carefully at where you pasted.

Reply:
Actually since you mentioned it in the PR description, it seems like it was actually intentional? In that case query.sql should _only_ contain queries, and no DELETE statements or other mutations.

Reply:
oh dear, let me fix that